schemas/parser-metadata/metadata.schema.json

```diff
@@ -0,0 +1,28 @@
+{
+  "$schema": "https://json-schema.org/draft/2020-12/schema",
+  "type": "object",
+  "title": "Parsed document metadata",
+  "properties": {
+    "parameters": {
+      "type": "object",
+      "properties": {
```

Comment:
The backlog parser is also currently populating a parameter called `INGESTER_OPTIONS` which we want to keep for the bulk parse

```json
        "INGESTER_OPTIONS": {
            "auto_publish": true,
            "source_document": {
                "format": "application/pdf",
                "file_hash": "b0f837c1a2df7d85983302781d3319410ecf6ac9b2c2aaee767d776f15ec7c24"
            }
        },
```